api.md

```diff
@@ -2767,6 +2767,10 @@ Specifically this sets the `timescaledb.restoring` GUC to `on` and stops any
 background workers which may have been performing tasks until the [`timescaledb_post_restore`](#timescaledb_post_restore)
 fuction is run following the restore. See [backup/restore docs][backup-restore] for more information.
 
+Note that you do not need so use this function when doing an upgrade
```

Comment:
should this be tip or warning?

Reply:
I think a warning is more appropriate. Changed.